dashboard/data/combine.py

```diff
@@ -6,48 +6,48 @@
 from dashboard.data.bmg_plate import Mode, get_activation_inhibition_zscore_dict
 
 
-# NOTE: to be removed
-def combine_assays(
-    dataframes: list[pd.DataFrame],
+def combine_assays_for_projections(
```

Comment:
duplicate method (it exists also in `preprocess.py`)

Reply:
Good point, sorry for that!